README.md
# RedK0d Clickable
 A partially clickable mod for the FC3 Planes

 Supported
 * Su-27  
 * J-11A  
 * Su-33  
 * Su-25T  
 * Su-25  
 * A-10A  
 * F-15C   

 # INSTALLATION 

* __Step 1:__ Unzip the contents of the archive in your "Saved Games\DCS\Mods\tech" directory.  


* __Step 2:__ Launch the game.  
 You can manage the activation of the module in the options  
 Lalt+C to click  
 Please report any issues in the dedicated section on Discord or Github  

## Concerning the F-15C it is a little more complex:  

* __Step 1:__ Download and Install Open Mod Manager Available [here](https://github.com/sedenion/OpenModMan/releases)  
* __Step 2:__ Open the Redk0d Clickable F-15C.omc file contained in CLICKABLE-FC3(version)\DATA\Open Mod Manager  
* __Step 3:__ Click on "Edit"  
* __Step 4:__ Click on "Target Destination Folder"   
* __Step 5:__ Enter your DCS installation directory  
* __Step 6:__ Click Apply  
* __Step 7:__ Depending if you have Standalone F-15C or FC3 highlight "F-15C Standalone" or "F-15C With FC3" and click "Install" or "Unistall" if you want to install or uninstall.  
* __Step 8:__ Fly  
 
 # SUPPORT US
3 ways to support us 

* [Our Store](https://redk0d.gumroad.com/l/fvkodo)   

* [Patreon](https://www.patreon.com/RedK0d)   

* [Paypal](https://www.paypal.com/donate/?hosted_button_id=8RA626VEJD2SC)   

 # DISCORD
Join us on [Discord](https://discord.com/invite/CrExr87dP9)

 # SPECIAL THANKS
 * __Tobsen__                               (Edm Exporter for Blender)  
 * __Grim Reapers,MatAWG and Sklang__       (YouTube content creators)  
 * __Aether,ImAPanzer and Emerald 467__     (For Discord Management)  
 * __CrunchZ08__                            (For his information on the F-15C systems)  
 * __Emerald 467 deadlyfishes and snowman__ (For tests and feedback)  
 * __Zeitgeist__                            (LowFidelityAircraftManual)  
 * __denissoliveira__                       (Help Images)  
 
 
 * __And for you all__                      (For your interest in this mod)  




CHANGELOG.md
# 23/04/2023 v1.1.1e-beta

Removed the popup

# 28/09/2022 v1.1.1b-beta

Changed the installation method.
    Look in the install file
F-15C
.Fixed Illumination Cocpit Issue
.Fixed Bingo Fuel Issue    
# 24/09/2022 v1.1.1a-beta
Su-25
.Fixed Salvo Control 
    Was not moving two notches at a time.
Su-25T
.Fixed misplacement of landing gear lever connector
    Was not moving

MiG-29S/A/G
.Fixed misplacement of landing gear lever connector
.Fixed bug about electro Electro-Optical System On/Off
.Fixed bug about Radar On/Off

F-15C
.Fixed throttle connectors location

Rework of the F-15C part.
The mod uses the connectors already provided in the cockpit and can thus use the animations.

Creation of a customized countermeasure system.
    .Choose, Chaff, Flares, or both, by operating a switch.
    .Two modes supported, Manual, or Semi Automatic.
    .Important, see the INSTALL and follow the instructions to make it work.

Creation of a lighting management system..
    .The LDG/TAXI light lever is now functional.
    .The navigation lights are adjustable in intensity, and can now flash.
    .The anti-collision lights are managed independently and are activated with their own switch.
    .The cockpit lighting can be activated by manipulating a switch, and is adjustable by manipulating a rotary.
    .You can now turn off the formation lights by turning the relevant knob to OFF.

Many other animations integrated. Most are cosmetic, but some others are usable.
Work still in progress

# 03/06/2022 v1.0.2c-beta

Added Help Images produced by @denissoliveira#9693 thanks to him.

All 
.Added_Message  Next/Previous Waypoint, Airfield now triggers a user message             

Su-27/J-11A
.Added Launch Permission Override
.Moved Radar Pulse Repeat Frequency Select
.Moved Electro-Optical System On/Off
.Added Receive Mode
.Removed HOTAS Target Unlock (Throttle)
.Added HOTAS Communication menu (Throttle)

Su-33
.Added Launch Permission Override
.Moved Radar Pulse Repeat Frequency Select
.Moved Electro-Optical System On/Off
.Added Receive Mode
.Removed HOTAS Target Unlock (Throttle)
.Added HOTAS Communication menu (Throttle)

MiG-29S/A/G
.Added Launch Permission Override

Su-25T
.Moved Canopy Lever 
    Now follows the canopy and lever
.Fixed Laser Ranger
    Was not working    

Su-25
.Moved Canopy Lever 
    Now follows the canopy and lever
.Removed HOTAS Autopilot - Transition To Level Flight Control
.Removed HOTAS Autopilot - Reset
.Added HOTAS Target Designator To Center
 
F-15C
.Added Illumination Cockpit
.Added HUD Color
.Fixed Gear Light Near/Far/Off Bug
       The command was executing twice
.Fixed T/O Trim
        Renamed "Autopilot Disengage" to "T/O Trim" and fixed the command
.Moved Navigation lights
.Added Anti-collision lights
        
# 21/05/2022 v1.0.2a-beta

Fixed issues
.Interference issue with All Modules
    Reworked mod logic. 
    The mod only starts if the aircraft is supported.

Su-27/J-11A
.Moved Scan Zone Left
.Moved Scan Zone Right
.Removed Scan Zone Up
.Removed Scan Zone Down
.Added Scan Zone Up/Down 

Su-33
.Moved Scan Zone Left
.Moved Scan Zone Right
.Removed Scan Zone Up
.Removed Scan Zone Down
.Added Scan Zone Up/Down 

MiG-29A/G/S
.Added Scan Zone Up/Down

Su-25T
.Removed HOTAS Target Lock (Handle)
.Added HOTAS Quick A-A to A-G Switch

Su-25
.Removed HOTAS Target Lock (Handle)
.Added HOTAS Quick A-A to A-G Switch
.Fixed ASP-17 Glass Up/Down (Only cycled down)

F-15C
.Added          Bingo Fuel Index CW/CCW
.Label_Change   Radar Off
            >   Radar On/Off 
.Label_Change   Parking Brakes
            >   Emergency Brakes 
                (Left Click):Engage  
                (Right Click):Disengage
.Added_Message  Emergency Brakes now triggers a user message             
A-10A
.Label_Change   Wheel Brake
            >   Emergency Brakes 
                (Left Click):Engage  
                (Right Click):Disengage 
.Added_Message  Emergency Brakes now triggers a user message                




# 12/05/2022 v1.0.1c-beta
Fixed issues
.Interference issue with F-16I SUFA Module
F-15C
.Added HOTAS Target Lock
.Added HOTAS Radar - Return To Search/NDTWS
.Added HOTAS Weapon Change
.Added HOTAS Weapon Release
.Added HOTAS Target Designator
.Added HOTAS Trim Controler

# 11/05/2022 v1.0.1b-beta
Su-25
.Fixed  Target Designator Issue
F-15C
.Fixed  Radar Scan Zone Up/Down Issue
.Label_Change   Fuel Quantity Selector  (Click) Fuel Quantity Selector (Scroll)
            >   Fuel Indicator  
                (Click):Quantity Test  
                (Scroll):Quantity Selector
.Label_Change   RWR Mode Select (Click) RWRSound Signals Volume Up/Down (Scroll)
            >   "RWR  
                (Click):Mode Select  
                (Scroll):Signals Volume Up/Down"
.Label_Change   Emergency Brake 
            >   Parking Brakes  
                (Left Click):Engage  
                (Right Click):Disengage
.Label_Change   Master Modes Select 
            >   Master Combat Mode 
                Beyond Visual Range
                Close Air Combat Vertical Scan 
                Close Air Combat Bore 
                Longitudinal Missile Aiming /FLOOD mode
.Fixed Master Combat Mode Issue                      

# 10/05/2022 v1.0.1-beta
F-15C
    .Removed Flight Clock Start/Stop/Reset
        [The F-15C does not support this command]
    .Fixed Emergency Brake
    .Fixed Master Modes Select
        [Need further testings]
    .Fixed Gear Light Near/Far/Off
    .Fixed Navigation Modes
        [Need further testings]
    .Fixed Radar On/Off
        [Need further testings]

# 09/05/2022 v1.0.0-beta

F-15C
    .Moved Engine Left Start
    .Moved Engine Right Start 
    .Added HUD Brightness Up/Down
    .Added Altimeter Pressure Increase/Decrease
    .Added Elapsed Time Clock Start/Stop/Reset
    .Added Flight Clock Start/Stop/Reset
    .Added Eject (3 times)
    .Added Emergency Brake
    .Added Fuel Dump
    .Added Weapons Jettison
    .Added Emergency Jettison
    .Added Fuel Tanks Jettison

entry.lua
declare_plugin("RedK0d Clickable",
{
	installed 	  	= 	true,
	dirName		  	= 	current_mod_path,
	displayName   	= 	"RedK0d Clickable",
	shortName	  	= 	"RedK0d Clickable",
	fileMenuName  	= 	"RedK0d Clickable",
	version			=	"v1.1.1e-beta",
	state		  	= 	"installed",
	developerName 	= 	"RedK0d",
	info		  	= 	"RedK0d Clickable",
	
	load_immediately = true,
	binaries	 = {},
	Skins	=
	{
		{
			name	= ("RedK0d Clickable"),
			dir		= "Skins"
		},
	},
	Options =
	{
		{
			name		= ("RedK0d Clickable"),
            nameId		= "RedK0d Clickable",
            dir			= "Options",
            CLSID		= "{FC3 CLICKABLE}"
		},
	},
	
})


local path 		= current_mod_path..'/Cockpit/Scripts/'
mount_vfs_texture_path  (current_mod_path ..  "/Textures")
mount_vfs_model_path    (current_mod_path ..  "/Shapes")

		

		
add_plugin_systems('CLICKABLE-FC3_Module','*',path,
	{
	
	--Alpha Implemented
	["J-11A"] 					= {enable_options_key_for_unit = 'J11a_enabled'},
	["Su-27"] 					= {enable_options_key_for_unit = 'Su27_enabled'},
	["Su-33"] 					= {enable_options_key_for_unit = 'Su33_enabled'},
	["Su-25T"] 					= {enable_options_key_for_unit = 'Su25t_enabled'},
	["MiG-29A"] 				= {enable_options_key_for_unit = 'Mig29a_enabled'},
	["MiG-29G"] 				= {enable_options_key_for_unit = 'Mig29g_enabled'},
	["MiG-29S"] 				= {enable_options_key_for_unit = 'Mig29s_enabled'},
	["Su-25"] 					= {enable_options_key_for_unit = 'Su25_enabled'},
	["A-10A"]					= {enable_options_key_for_unit = 'A10a_enabled'},
	["F-15C"]					= {enable_options_key_for_unit = 'F15c_enabled'},
	["TEST"]					= {enable_options_key_for_unit = 'Version'},
	}
)

plugin_done()
